client/my-sites/hosting/staging-site-card/index.js

```diff
@@ -198,16 +211,16 @@ export const StagingSiteCard = ( { currentUserId, disabled, siteId, siteOwnerId,
 	const getManageStagingSiteContent = () => {
 		return (
 			<>
-				<p>
-					{ translate( 'Your staging site is available at {{a}}%(stagingSiteName)s{{/a}}.', {
-						args: {
-							stagingSiteName: stagingSite.url,
-						},
-						components: {
-							a: <a href={ stagingSite.url } />,
-						},
-					} ) }
-				</p>
+				<p>{ translate( 'Your staging site is available at:' ) }</p>
+				<SiteRow>
+					<SiteIcon siteId={ stagingSite.id } size={ 40 } />
```

Comment:
What happens when the user doesn't have access to the staging site?

Reply:
Hi @danielbachhuber  - I took a look and if user does not have access to the site, they will get access error:

```
const getAccessError = () => {
		return (
			<Notice status="is-error" showDismiss={ false }>
				<div data-testid="staging-sites-access-message">
					{ translate(
						'Unable to access the staging site {{a}}%(stagingSiteName)s{{/a}}. Please contact the site owner.',
						{
							args: {
								stagingSiteName: stagingSite.url,
							},
							components: {
								a: <a href={ stagingSite.url } />,
							},
						}
					) }
				</div>
			</Notice>
		);
	};
```

This is what it looks like:

<img width="1199" alt="Screenshot 2023-04-05 at 2 24 46 PM" src="https://user-images.githubusercontent.com/25575134/230080049-edaa9cd1-7086-47d4-bb63-19ad30d85a3a.png">

I have not touched that part of the code though so the functionality of this PR does not modify anything related to user permissions.

Reply:
@katinthehatsite Cool, that works too. Thanks for the detailed response 😊